pkg/serve/config.go

```diff
@@ -226,6 +229,17 @@ func NewConfig() (*Config, error) {
 	} else {
 		c.RPCGasCap = big.NewInt(0)
 	}
+	if sdTimeout := viper.GetString("ethereum.stateDiffTimeout"); sdTimeout != "" {
+		var err error
+		if c.StateDiffTimeout, err = time.ParseDuration(sdTimeout); err != nil {
+			return nil, err
+		}
+	} else {
+		c.StateDiffTimeout = ethServerShared.DefaultStateDiffTimeout
+	}
+	if c.StateDiffTimeout < 0 {
+		return nil, errors.New("ethereum.stateDiffTimeout < 0")
```

Comment:
Maybe in this case we should revert to the default and throw a `Info/Warn` message? Or is it better to throw an error so the user is made abundantly clear they provided a faulty configuration that isn't being used?

Reply:
I prefer the latter, since this should rarely happen